packit_service/worker/helpers/testing_farm.py

```diff
@@ -554,6 +589,21 @@ def check_comment_pr_argument_and_report(self) -> bool:
 
         return True
 
+    def check_comment_identifier_and_report(self) -> bool:
```

Comment:
maybe I am missing something, but is this method used somewhere?

Reply:
I am thinking about what would be the best UX here, probably rather only adding a comment with the same description instead of setting checks for all test jobs :thinking: WDYT?

But it's a little bit more tricky where to place this check. I can take a look later

Reply:
Ye that sounds reasonable to me. So I will remove the method from the PR for now.